CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# Version 2.1.2
+## Features
+- New transaction list UI;
+- New transaction detail UI.
+
+## Bug fixes
+- Fixed a bug that cause the home to reload the transaction list after showing the detail of a transaction;
+- Fixed a bug that prevent the user to write their mnemonic.
```

Comment:
Since we're changing UI it's better to use `v2.2.0` instead

```suggestion
# Version 2.2.0
## Features
- New transaction list UI
- New transaction details UI

## Bug fixes
- Fixed a bug that caused the home to reload the transaction list after showing the detail of a transaction
- Fixed a bug that prevented the user to write their mnemonic
```